src/plugins/intel_cpu/src/nodes/ctc_greedy_decoder_seq_len.cpp

```diff
@@ -85,15 +85,15 @@ void CTCGreedyDecoderSeqLen::execute(dnnl::stream strm) {
 
     int blankIndex = C - 1;
     if (inputShapes.size() > BLANK_INDEX)
-        blankIndex = (reinterpret_cast<const int  *>(getParentEdgeAt(BLANK_INDEX)->getMemoryPtr()->getData()))[0];
+        blankIndex = (getSrcDataAtPortAs<const int >(BLANK_INDEX))[0];
 
     size_t workAmount = 0;
     for (size_t b = 0; b < B; b++) {
         if (sequenceLengths[b] > static_cast<int>(T)) {
             std::string errorMsg = errorPrefix
                                    + ". Sequence length " + std::to_string(sequenceLengths[b])
                                    + " cannot be greater than according decoded classes dimension size "
-                                   + std::to_string(getChildEdgesAtPort(DECODED_CLASSES_INDEX)[0]->getMemory().getStaticDims()[1]);
+                                   + std::to_string(getChildEdgeAt(DECODED_CLASSES_INDEX)->getMemory().getStaticDims()[1]);
```

Comment:
```suggestion
                                   + std::to_string(getDstMemoryAtPort(DECODED_CLASSES_INDEX).getStaticDims()[1]);
```
And this is not the only one such occurrence in the PR. May be yet another regex substitution pass is required?

Reply:
Initially I didn't want to cover getMemory() calls, since the main focus was accessing MemoryPtr for oneDNN nodes and raw memory ptr for the rest of the nodes.
But I think I can cover the getMemory() cases as well

Reply:
Discussed. More appropriate change will be to have two version getSrcMemoryPtrAtPort and getSrcMemoryAtPort, and use them depending on the context. This requires careful manual changes in many code places.